### ISSUE TITLE
Decorative images require ARIA tags

### ISSUE THREAD

**Issue by VirtualDOMinic** [thejoefriel/thejoefriel.github.io#1]
Possibly useful link: https://developer.paciellogroup.com/blog/2012/05/html5-accessibility-chops-hidden-and-aria-hidden/ 

Should be able to use `aria-hidden=true` to ensure these are hidden in the AOM
